modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,184 @@
+
+External clients can trigger processing in a Mule flow through a number of
+communication protocols and methods, such as JMS, HTTP, FTP, JDBC, or File.
+Mule translates these communication protocols and methods into a standard
+message format, which passes through the flow's processors.
+
+Sources in Mule can connect to specific external sources, either through
+a standard protocol or a third-party API. It is also possible to set a Scheduler
+component. Some schedulers can poll for specific changes to external resources,
+such as new files or table rows in an external resource. Examples of listeners
+and connector operations that can trigger a flow include:
+
+* HTTP, JMS, VM listeners in their associated connectors
+* On Table Row operation in the Database connector
+* On New or Updated File operation in the File and FTP connectors
+* Scheduler
+
+Note that you can also create your own source using the
+xref:mule-sdk::index.adoc[Mule SDK].
+
+=== Processors
+
+Once a flow is triggered through a source component, subsequent components process
+the data as it travels through the flow. By default, each processor
+that receives a Mule event returns a new Mule message, typically with a set
+of attributes and the message payload that the processor returns. The processor
+passes on the new message as output to the next processor in the flow.
+
+Processors available to Mule applications include:
+
+* Components from modules and connectors. Examples include operations that read
+from and write to an external resource and that validate data in the Mule application.
++
+Some operations can make client requests to external resources and services
+(including external databases and systems, such as Salesforce, Workday,
+ServiceNow, and many others) and to other Mule applications. Others can run your
+custom code, support OAuth configurations, and handle communication through
+asynchronous queues, for example. A variety of other operations are
+available.
++
+* Core components can route data, perform data transformations, handle errors
+that might occur when processing the event, and perform other tasks in a
+Mule application.
++
+Transformers (such as the Transform Message, Set Variable, and others) are
+key to exchanging data between nodes. Transformers enable Mule to convert
+message data in the Mule event to a format that another application or service can
+read.
++
+Mule also enables content enrichment of messages (through
+xref:target-variables.adoc[Target Parameters]) so that you can retrieve
+additional data and attach it to the message.
+
+Note that you can also create your own Event processors using the
+xref:mule-sdk::index.adoc[Mule SDK].
+
+== Security
+
+You can provide security to your Mule applications by encrypting properties, configuring
+secure communications over TLS, setting up authentication over OAuth 2.0, and
+providing cryptographic and other capabilities, such as FIPS compliance.
+
+* xref:securing.adoc[Security]
```

Comment:
This should be a sentence rather than a single list item: "For more information about...".